releases/exact_version.go

```diff
@@ -27,6 +27,7 @@ type ExactVersion struct {
 	Version    *version.Version
 	InstallDir string
 	Timeout    time.Duration
+	Enterprise EnterpriseOptions
```

Comment:
```suggestion
	Enterprise EnterpriseOptions
```
As mentioned we could turn this into a pointer, which could be equivalent to `Enterprise == true` but slightly simpler.